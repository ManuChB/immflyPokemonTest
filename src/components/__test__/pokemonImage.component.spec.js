import React from 'react';
import {
    configure,
    shallow
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import PokemonImage from '../pokemonImage.component';

configure({
    adapter: new Adapter()
});


describe('PokemonImage component', () => {
    const name =  'eevee';

    it('renders as expected', () => {
        //when
        const wrapper = shallow( < PokemonImage pokemonName = { name } > </PokemonImage>);
        //expect
        expect(wrapper).toMatchSnapshot();
    });
});