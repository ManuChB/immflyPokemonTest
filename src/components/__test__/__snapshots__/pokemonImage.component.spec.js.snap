// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonImage component renders as expected 1`] = `ShallowWrapper {}`;
